service/history/queues/executable.go

```diff
@@ -196,7 +197,7 @@ func (e *executableImpl) HandleErr(err error) (retErr error) {
 	// TODO remove this error check special case
```

Comment:
Could we remove this check?

Reply:
I don't know :)
@yycptt Should I remove it or do you want to do it later in your PR?

Reply:
I will do it later after merging active & standby queue processor.